sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/src/SchemaRegistryAvroEncoder.cs

```diff
@@ -306,7 +312,7 @@ private static DatumReader<object> GetReader(Schema writerSchema, Schema readerS
             }
```

Comment:
Should we consider avoiding the array for `schemaIdBytes` on L309 by using a span?

```csharp
ReadOnlySpan<byte> schemaIdBytes = memory.Slice(RecordFormatIndicatorLength, SchemaIdLength).Span;
schemaId = Utf8Encoding.GetString(schemaIdBytes);
```

Reply:
GetString does not have an overload that takes a span

Reply:
bah. I forgot to down-level from net6.   My bad; ignore me.